flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java

```diff
@@ -50,9 +51,11 @@ public DorisOptions(
             String password,
             String tableIdentifier,
             String jdbcUrl,
-            boolean redirect) {
+            boolean redirect,
+            String charsetEncoding) {
```

Comment:
It seems that this parameter is only used by SchemaChangeManager. It is recommended to add it to the constructor of SchemaChangeManager separately.

Reply:
Maybe other will use this parameter, such as flink cdc doris pipeline

Reply:
This parameter is only used by SchemaChangeManager currently, but I'm not sure if other modules have the same problem, so wouldn't it be better to expose it here?

Reply:
BTW, what time about next release

Reply:
Doris currently only supports utf8 encoding～

Reply:
Fine